### PR TITLE
feat(scss): adding colors-classes

### PR DESCRIPTION
**Pull request by tigredonorte**

### CODE REVIEW

File: .czrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "@endemolshinegroup/cz-jira-smart-commit",
+  "@endemolshinegroup/cz-jira-smart-commit": {
+    "pullRequestsOnly": true,
+    "scopes": {
+      "scss": "For changes in scss files"
```
Comment (oliveira-gust) [inline .czrc]: Nice! You could take a look at the other repositories and add default scopes from there too, like "cz" or "dependencies"

File: scss/_colors-classes.scss [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+  }
+
+  &.benchmark {
+    background-color: rgba(map-get($asset-colors-primary, benchmark), $opacity) !important;
+  }
+}
+
+%asset-bg {
+  @include generate-asset-bg(1);
+  color: #fff;
```
Comment (oliveira-gust) [inline scss/_colors-classes.scss]: Could be imported from colors

File: scss/_colors-classes.scss [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+}
+
+.asset-bg {
+  @extend %asset-bg;
+}
+
+.asset-bg-ribbon {
+  $opacity: 1;
+  $opacity2: 0.15;
```
Comment (oliveira-gust) [inline scss/_colors-classes.scss]: Maybe these guys deserve more descriptive names... what do you think? 🤔

File: scss/_colors.scss [hunk 4]
```diff
@@ -29,3 +29,34 @@ $medium-gray: #cacbcc;
...
+  currency: $currency-secondary,
+  default: $gorila-secondary,
+  fixed_income: $fixed-income-secondary,
+  multimarket: $multimarket-secondary,
+  stocks: $stocks-secondary,
+  tangible: $tangible-secondary,
+);
+
+$neutral-colors: ( 
+    darkest: #000000,
```
Comment (oliveira-gust) [inline scss/_colors.scss]: $black

File: scss/_colors.scss [hunk 5]
```diff
@@ -29,3 +29,34 @@ $medium-gray: #cacbcc;
...
+  cash: $cash-primary,
+  currency: $currency-primary,
+  default: $gorila-primary,
+  fixed_income: $fixed-income-primary,
+  multimarket: $multimarket-primary,
+  stocks: $stocks-primary,
+  tangible: $tangible-primary,
+);
+$asset-colors-secondary: (
+  benchmark: #a3a5a6,
```
Comment (oliveira-gust) [inline scss/_colors.scss]: Maybe these benchmark colors can be new variables? Are they used elsewhere?

File: scss/_colors.scss [hunk 6]
```diff
@@ -29,3 +29,34 @@ $medium-gray: #cacbcc;
...
+);
+
+$neutral-colors: ( 
+    darkest: #000000, 
+    darker: $extra-dark-gray, 
+    dark: #555b5b, 
+    normal: $dark-gray, 
+    light: #d6d4d5, 
+    ligher: #f0f0f0, 
+    lightest: #ffffff
```
Comment (oliveira-gust) [inline scss/_colors.scss]: $white

File: scss/_colors.scss [hunk 7]
```diff
@@ -29,3 +29,34 @@ $medium-gray: #cacbcc;
...
+  cash: $cash-secondary,
+  currency: $currency-secondary,
+  default: $gorila-secondary,
+  fixed_income: $fixed-income-secondary,
+  multimarket: $multimarket-secondary,
+  stocks: $stocks-secondary,
+  tangible: $tangible-secondary,
+);
+
+$neutral-colors: (
```
Comment (oliveira-gust) [inline scss/_colors.scss]: I think this could be moved up to stay closer to the GRAY SCALE variables

File: .czrc (hunk 1 above)
Comment (tigredonorte) [reply .czrc]: ok

File: package.json [hunk 8]
```diff
@@ -1,11 +1,13 @@
...
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Gorila invest package to hold styles like colors, mixins, variables, etc for all stylesheets technology (SASS, Stylus, CSS-in-JS, etc.)",
   "main": "index.js",
   "scripts": {
     "lint:scss": "stylelint scss/**/*.scss",
-    "lint:scss:fix": "stylelint scss/**/*.scss --fix"
+    "lint:scss:fix": "stylelint scss/**/*.scss --fix",
+    "build": "sass --update scss:css",
+    "dev": "sass --watch scss:css"
```
Comment (guilhermejcgois) [inline package.json]: Nice, thanks

**Pull request merged**